sklearn/compose/tests/test_column_transformer.py

```diff
@@ -516,29 +516,43 @@ def predict(self, X):
 def test_make_column_transformer():
     scaler = StandardScaler()
     norm = Normalizer()
-    ct = make_column_transformer(('first', scaler), (['second'], norm))
+    ct = make_column_transformer((scaler, 'first'), (norm, ['second']))
     names, transformers, columns = zip(*ct.transformers)
     assert_equal(names, ("standardscaler", "normalizer"))
     assert_equal(transformers, (scaler, norm))
     assert_equal(columns, ('first', ['second']))
 
+    # XXX remove in v0.22
+    with pytest.warns(DeprecationWarning,
+                      match='`make_column_transformer` now expects'):
+        make_column_transformer(('first', scaler))
+
+    with pytest.warns(DeprecationWarning,
+                      match='`make_column_transformer` now expects'):
+        make_column_transformer(('first', 'drop'))
+
+    with pytest.warns(DeprecationWarning,
+                      match='`make_column_transformer` now expects'):
+        make_column_transformer(('passthrough', 'passthrough'),
+                                ('first', 'drop'))
```

Comment:
Please add a check that the output of `fit_transform` with this tuple order is the same as the output of `fit_transform` with the non deprecated tuple order.